### DISCUSSION TITLE
listening rich presence

### DISCUSSION THREAD

**Discussion by theycallmerubik** [jojo2357/Music-Discord-Rich-Presence#54] [Q&A]
is it possible to change the playing ..... to listening in codes?
cuz we are not playing the music we are (listening) to it

**Comment by jojo2357** [comment]
[This](https://github.com/Lachee/discord-rpc-csharp) is the DRP library I use. If you  can make it configureable to say listening and not playing, then do let me know. At last i checked it is not possible because it is using the game sdk which focuses on `playing` and not listening.
